bot/cogs/core/voting.py

```diff
@@ -4,13 +4,76 @@
 import arrow
 import discord
 from discord.ext import commands
+from discord.utils import format_dt
 
+
+from bot.utils.ctx import CustomContext
 from bot.cogs.core.database import Database
 from bot.cogs.core.quests import Quests
+from bot.utils.misc import get_user_and_lang_from_loc
 from bot.villager_bot import VillagerBotCluster
 from common.models.topgg_vote import TopggVote
 
 
+class VoteReminderView(discord.ui.View):
+    def __init__(
+        self,
+        *,
+        bot: VillagerBotCluster,
+        loc: CustomContext | commands.Context | discord.User,
```

Comment:
I originally used a arg called loc that could've been any of these because I needed to support all of them, here I think we'll only ever just have a user object, so we can probably just replace this with `user: discord.User`